_data/chains/eip155-41.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Telos EVM Testnet",
+  "chain": "TLOS",
+  "network": "testnet",
+  "rpc": [
+    "https://testnet.telos.net/evm"
+  ],
+  "faucets": [
+    "https://app.telos.net/testnet/developers",
+    "https://api.telos.net/v1/docs/index.html#/testnet/get_v1_testnet_evmFaucet__evmAddress_"
```

Comment:
please remove this one - this will surface to end users and we cannot present them with API documentation. Maybe we find another field for it